packages/mdc-elevation/README.md

```diff
@@ -94,7 +94,7 @@ Mixin | Description
 `mdc-elevation-overlay-common` | Called once per application to setup the universal elevation overlay styles
 `mdc-elevation-shadow($box-shadow)` | Sets the `box-shadow` of the closest parent selector
 `mdc-elevation-overlay-surface-position` | Sets the positioning of the overlay's surface element so that the overlay can be appropriately centered
-`mdc-elevation-overlay-size($width, $height: $width)` | Sets the width and height of the elevation overlay
+`mdc-elevation-overlay-dimensions($width, $height: $width, $has-content-sizing: false)` | Sets the dimensions of the elevation overlay
```

Comment:
Looks like $has-content-sizing defaults to true

Reply:
Good catch, thanks!